tools/build_config/Jenkinsfile

```diff
@@ -355,15 +372,18 @@ pipeline {
       }
       steps {
         script {
+
           if (isUnix()) {
             sh '''
+              export PATH=${PATH}:~/.local/bin &&
```

Comment:
What's the path addition here for? Do we need them for the pip installed packages? A comment may be useful

Reply:
if we need to explicitly include this path in multiple build steps can you add it as a constant to help with maintainability.

There is also an inconsistency in the ordering of the `PATH` updates and the `module load`s : before in 'build' and 'analyse' steps, after in the 'build-docs' step